.github/workflows/main.yml

```diff
@@ -45,11 +45,22 @@ jobs:
         toolchain:
           - "gcc"
           - "clang"
+        dds_vendor:
```

Comment:
CI changes are OK, but lets leave this for a follow-up. For instance, I have to check how much buildbuddy remote cache we use and not overspend on it.